lib/rules/sort-object-props.js

```diff
@@ -1,5 +1,7 @@
 "use strict";
 
+var checkIgnoredTypes = require("./check-ignored-types");
```

Comment:
I think you should move this file up a level in the hierarchy (`lib/`), so it is not confused for an eslint rule.